rpc/params/status.go

```diff
@@ -147,6 +147,7 @@ type ApplicationStatus struct {
 	CharmVersion     string                     `json:"charm-version"`
 	CharmProfile     string                     `json:"charm-profile"`
 	CharmChannel     string                     `json:"charm-channel,omitempty"`
+	CharmRev         int                        `json:"charm-rev,omitempty"`
```

Comment:
We'll need to wire `CharmRev` up for this to actually work.

Reply:
Apply the following diff, I'd expect to see some test fallout once we do this...

```
diff --git a/apiserver/facades/client/client/status.go b/apiserver/facades/client/client/status.go
index 0c51848213..331b97b480 100644
--- a/apiserver/facades/client/client/status.go
+++ b/apiserver/facades/client/client/status.go
@@ -1392,6 +1392,7 @@ func (context *statusContext) processApplication(application *state.Application)
                Charm:        applicationCharm.URL(),
                CharmVersion: applicationCharm.Version(),
                CharmProfile: charmProfileName,
+               CharmRev:     applicationCharm.Revision(),
                CharmChannel: channel,
                Base: params.Base{
                        Name:    base.OS,
```

Reply:
Diff applied.

Reply:
Thanks, let's see what the fallout is once it's built.

Reply:
@amandahla can you run `make rebuild-schema` from the root of `juju/juju` project. As we've changed the status, we need to make others aware of that change.

Reply:
Here is the fallout, just 9 tests by the looks of it. https://pastebin.ubuntu.com/p/cT2TTT5zJP/

Reply:
Thanks, from what I understand, I only needed to update two files. I hope that's correct.